nexus/tests/config.test.toml

```diff
@@ -10,7 +10,7 @@ session_absolute_timeout_minutes = 1440 # 24 hours
 
 # List of authentication schemes to support.
 [authn]
-schemes_external = ["spoof", "session_cookie"]
+schemes_external = ["spoof", "session_cookie", "access_token"]
```

Comment:
This is necessary to make tokens actually work in the integration tests, so we can test that they work and that they expire.